mkdocs.yml

```diff
@@ -132,6 +131,8 @@ nav:
     - contributing/index.md
     - Reporting a bug: contributing/reporting-a-bug.md
     - Adding translations: contributing/adding-translations.md
+    - Community:
+      - Community forums: https://forums.slimdevices.com/
```

Comment:
A single item doesn't merit its own section, does it?

And while you're at it: could you please label the landing page something like "Overview", as without any label we'd end up with a title "Contributing", followed by a first item "Contributing", which looks odd.